src/pages/home/report/ReportActionCompose.js

```diff
@@ -676,17 +869,30 @@ class ReportActionCompose extends React.Component {
                                             style={[styles.textInputCompose, this.props.isComposerFullSize ? styles.textInputFullCompose : styles.flex4]}
                                             maxLines={this.state.maxLines}
                                             onFocus={() => this.setIsFocused(true)}
-                                            onBlur={() => this.setIsFocused(false)}
+                                            onBlur={() => this.setIsBlured(false)}
                                             onPasteFile={displayFileInModal}
                                             shouldClear={this.state.textInputShouldClear}
                                             onClear={() => this.setTextInputShouldClear(false)}
-                                            isDisabled={isComposeDisabled || isBlockedFromConcierge || this.props.disabled}
+                                            isDisabled={isComposeDisabled || isBlockedFromConcierge}
                                             selection={this.state.selection}
                                             onSelectionChange={this.onSelectionChange}
                                             isFullComposerAvailable={this.state.isFullComposerAvailable}
                                             setIsFullComposerAvailable={this.setIsFullComposerAvailable}
                                             isComposerFullSize={this.props.isComposerFullSize}
+                                            isSmallScreenWidth={this.props.isSmallScreenWidth}
                                             value={this.state.value}
+                                            onLayout={({
```

Comment:
NAB but I feel like this destructuring is hurting readability:

```suggestion
onLayout={e => this.setState({composerHeight: e.nativeEvent.layout.height})}
```